### PR TITLE
Fix prepare release

### LINKED ISSUE #1429: Fix prepare-release
When prepare-release was converted to a compiled program, the indentation of the new release body was messed up, so it's created mostly as a code block.

Also, somehow when I first committed the cross-referencing of the milestone and release body issues, I elided the `?` in the named capture group (I swear is was there when I was testing), so no issue numbers are detected in the release body.
Even if it was there, the group would miss multi-issues like `(#1234, #1235)`.

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #1429

### CODE REVIEW

File: tools/FakeItEasy.PrepareRelease/Program.cs [hunk 1]
```diff
@@ -102,10 +102,18 @@ public static async Task<IList<Issue>> GetIssuesInMilestone(this IGitHubClient g
...
             var issuesReferencedFromRelease = new HashSet<string>();
-            foreach (Match match in Regex.Matches(release.Body, @"\(#(<issueNumber>[0-9]+)\)"))
+            foreach (Match match in Regex.Matches(release.Body, @"\((?<issueNumbers>#[0-9]+((, )#[0-9]+)*)\)"))
             {
-                issuesReferencedFromRelease.Add(match.Groups["issueNumber"].Value);
+                var issueNumbers = match.Groups["issueNumbers"].Value;
+                foreach (var issueNumber in issueNumbers.Split(new char[] { '#', ' ', ',' }, StringSplitOptions.RemoveEmptyEntries))
+                {
+                    issuesReferencedFromRelease.Add(issueNumber);
+                }
```
Comment (thomaslevesque) [inline tools/FakeItEasy.PrepareRelease/Program.cs]: Nice touch!

**Comment by thomaslevesque** [comment]
Thanks @blairconrad!

File: tools/FakeItEasy.PrepareRelease/Program.cs (hunk 1 above)
Comment (blairconrad) [reply tools/FakeItEasy.PrepareRelease/Program.cs]: Oh? Thanks. I was frustrated that I couldn't just suck the values out with a regex. Ah well, it works. Probably overkill for Sprache!

**Comment by blairconrad** [comment]
Thanks for the merge!

**Pull request merged**